src/Metadata/AssemblyMetadata.Types.cs

```diff
@@ -79,7 +79,11 @@ private EntityHandle ResolveTypeReference(Type type)
 
         public bool IsGenericTypeSpec(Type type)
         {
+#if NETSTANDARD || NET46
+            return type.IsGenericMethodParameter() || type.IsGenericParameter || (type.IsGenericType && !type.IsGenericTypeDefinition);
```

Comment:
See remark above: we should keep the extension method call for .NET Core as well and drop the `#if`.

Reply:
Done.